devmock/endpoints.py

```diff
@@ -118,23 +119,23 @@ def complete_order(id: str, order: Order):
 def fail_order(id: str, order: Order):
     current_order = load_order(id)
     current_order["reason"] = order.reason
-    current_order["status"] = "Failed"
+    current_order["status"] = "failed"
     save_order(current_order)
     agreement = load_agreement(current_order["agreement"]["id"])
-    agreement["status"] = "Active"
+    agreement["status"] = "active"
     save_agreement(agreement)
     return current_order
 
 
 @router.post("/commerce/orders/{id}/query")
 def inquire_order(
     id: str,
-    template: Annotated[dict, Body()],
+    templateId: Annotated[str, Body()],
     parameters: Annotated[dict, Body()],
 ):
     order = load_order(id)
     order["parameters"] = parameters
-    order["template"] = template
+    order["templateId"] = template
```

Comment:
do the team have a bug for that ? According to docs is as it was before :)

Reply:
I think yes, but that's the current implementation. If it was changed --> I'll revert it